tutorials/rendering/multiple_resolutions.rst

```diff
@@ -132,39 +139,48 @@ Stretch settings are located in the project settings and provide several options
 Stretch Mode
 ^^^^^^^^^^^^
 
-The **Stretch Mode** setting defines how the base size is stretched to fit
-the resolution of the window or screen. The animations below use a "base
-size" of just 16×9 pixels to demonstrate the effect of different stretch
-modes. A single sprite, also 16×9 pixels in size, covers the entire viewport,
-and a diagonal :ref:`Line2D <class_Line2D>` is added on top of it:
+The **Stretch Mode** setting defines how the base size specified in the
+Project Settings` **Display** section is stretched to fit the resolution 
+of the window. The animations below use a base size of just 16×9 pixels to
+demonstrate the effect of different stretch modes. A single sprite, also 
+16×9 pixels in size, covers the entire viewport, and a diagonal 
+:ref:`Line2D <class_Line2D>` is added on top of it:
 
 .. image:: img/stretch_demo_scene.png
 
 .. Animated GIFs are generated from:
 .. https://github.com/ttencate/godot_scaling_mode
 
--  **Stretch Mode = Disabled** (default): No stretching happens. One
-   unit in the scene corresponds to one pixel on the screen. In this
-   mode, the **Stretch Aspect** setting has no effect.
+-  **Stretch Mode = Disabled** (default): No stretching happens. The
+   :ref:`Viewport <class_Viewport>` is resized from the base size to the 
+   window's resolution, so that one unit in the scene corresponds to one
+   pixel in the window. In this mode, the **Stretch Aspect** setting has
+   no effect.
 
    .. image:: img/stretch_disabled_expand.gif
 
--  **Stretch Mode = Canvas Items**: In this mode, the base size specified in
-   width and height in the project settings is
-   stretched to cover the whole screen (taking the **Stretch Aspect**
-   setting into account). This means that everything is rendered
-   directly at the target resolution. 3D is unaffected,
-   while in 2D, there is no longer a 1:1 correspondence between sprite
-   pixels and screen pixels, which may result in scaling artifacts.
+-  **Stretch Mode = Canvas Items**: The root :ref:`Viewport <class_Viewport>`
+   is resized from the base size to the window's resolution (taking the
+   **Stretch Aspect** setting into account). Each CanvasItem is then
+   scaled by the same amount as the base size was scaled.  For example, 
+   if the base size is 16x9 and the window is 160x180, each CanvasItem
+   will be stretched by a factor of 10 horizontally and 20 vertically.
```

Comment:
>  For example, if the base size is 16x9 and the window is 160x180, each CanvasItem will be stretched by a factor of 10 horizontally and 20 vertically.

I don't think this is true in every case, and it seems to directly contradict the gif just below it? It would be true if `Stretch Aspect` was `ignore`.